app/steps/ui/payment/breakdown/index.js

```diff
@@ -21,7 +21,8 @@ class PaymentBreakdown extends Step {
 
         ctx.copies = this.createCopiesLayout(formdata);
         ctx.applicationFee = fees.applicationfee;
-        ctx.total = Number.isInteger(fees.total) ? fees.total : parseFloat(fees.total).toFixed(2);
+        ctx.total = Number.isInteger(fees.total) ? fees.total : parseFloat(fees.total)
+            .toFixed(2);
```

Comment:
There isn't a change here, can you remove the line break, just in case this causes conflict errors later on.